Paws4Me/Paws4Me/Controllers/SignInViewController.swift

```diff
@@ -6,53 +6,96 @@
     @IBOutlet weak var txtUsername: UITextField!
     @IBOutlet weak var txtPassword: UITextField!
     @IBOutlet weak var btnSignIn: UIButton!
     @IBOutlet weak var btnRegister: UIButton!
-    
     var bLoggedIn = false
-    
+    let bottomLine = CALayer()
+    let bottomLine2 = CALayer()
     override func viewDidLoad() {
         super.viewDidLoad()
-        
-        btnSignIn.layer.cornerRadius = 10
-        btnRegister.layer.borderWidth = 1.0
-        btnRegister.layer.cornerRadius = 10
-        btnRegister.layer.borderColor = UIColor.init(red: 55/255,green: 21/255,blue: 67/255,alpha: 1).cgColor
-        
-        let bottomLine = CALayer()
-        
-        let bottomLine2 = CALayer()
-        
-        bottomLine.frame = CGRect(x: 0, y: txtUsername.frame.height - 2, width: txtUsername.frame.width , height: 2)
-        
-        bottomLine2.frame = CGRect(x: 0, y: txtPassword.frame.height - 2, width: txtPassword.frame.width , height: 2)
-        bottomLine.backgroundColor = UIColor.init(red: 55/255,green: 21/255,blue: 67/255,alpha: 1).cgColor
-        
-        bottomLine2.backgroundColor = UIColor.init(red: 55/255,green: 21/255,blue: 67/255,alpha: 1).cgColor
-        
-        txtUsername.borderStyle = .none
-        txtPassword.borderStyle = .none
-        txtUsername.layer.addSublayer(bottomLine)
-        txtPassword.layer.addSublayer(bottomLine2)
-        
+        btnSignIn.addCornerRadius(btnSignIn)
+        btnSignIn.addCornerRadius(btnRegister)
+        btnSignIn.changeBorderLook(btnRegister)
+        addUnderline(txtUsername, txtPassword, bottomLine: bottomLine, bottomLine2: bottomLine2)
     }
-
-    @IBAction private func btnSignInClick(_ sender: Any) {
-        let username = txtUsername.text
-        let password = txtPassword.text
-        
+    @IBAction private func btnSignInClick(_ sender: UIButton!) {
+        guard let username = txtUsername.text else { return }
+        guard let password = txtPassword.text else { return }
+        let bValidCredentials = isValidCredentials(username: username, password: password)
+        var errorMessage = ""
+    breakIf:
+        if bValidCredentials {
+            // resetBorderBoth(txtUsername, txtPassword, bottomLine: bottomLine, bottomLine2: bottomLine2)
+            performSegue(withIdentifier: "signInViewController", sender: self)
+        } else if !bValidCredentials {
+            errorMessage = "Incorrect Username or Password"
+            addErrorBorderBoth(txtUsername, txtPassword, bottomLine: bottomLine, bottomLine2: bottomLine2)
+            break breakIf }
+        displayErrorAlert(alertTitle: "Invalid credentials.",
+                          alertMessage: errorMessage,
+                          alertActionTitle: "Try again" ,
+                          alertDelegate: self)
+    }
+    func isValidCredentials(username: String, password: String) -> Bool {
         if username == "Admin" && password == "TestPass123" {
```

Comment:
This can be simplified to 

```
func isValidCredentials(username: String, password: String) -> Bool {
   return username == "Admin" && password == "TestPass123"
}
```